Makefile

```
@@ -100,11 +101,11 @@ operator: generate
 
 .PHONY: operator-image
 operator-image: generate
-	docker build -f build/Dockerfile . -t $(OPERATOR_IMG)
+	$(CONTAINER_RUNTIME) build -f build/Dockerfile . -t $(OPERATOR_IMG)
 
 .PHONY: operator-push
 operator-push:
-	docker push ${OPERATOR_IMG}
+	$(CONTAINER_RUNTIME) push $(PUSH_OPTION) ${OPERATOR_IMG}
```

Apologies, one last nit. 
```suggestion
	$(CONTAINER_RUNTIME) push $(PUSH_OPTIONS) ${OPERATOR_IMG}
```
Seems better? I think this can pass multiple options to push right? Otherwise lgtm

---

Ahh yeah. No problem. Let me update this because it's in multiple places.